packages/fxa-settings/src/pages/Signin/SigninRecoveryMethod/Phone/ConfirmRecoveryCode/index.test.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { renderWithLocalizationProvider } from 'fxa-react/lib/test-utils/localizationProvider';
+import { screen, fireEvent, waitFor } from '@testing-library/react';
+import ConfirmRecoveryCode from './index';
+
+describe('ConfirmRecoveryCode', () => {
+  const mockVerifyCode = jest.fn(() => Promise.resolve());
+  const mockResendCode = jest.fn(() => Promise.resolve());
+  const mockClearBanners = jest.fn();
+  const mockSetErrorMessage = jest.fn();
+
+  const defaultProps = {
+    clearBanners: mockClearBanners,
+    maskedPhoneNumber: '••••••1234',
+    errorMessage: '',
+    setErrorMessage: mockSetErrorMessage,
+    verifyCode: mockVerifyCode,
+    resendCode: mockResendCode,
+  };
+
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it('renders as expected', async () => {
+    renderWithLocalizationProvider(<ConfirmRecoveryCode {...defaultProps} />);
+
+    expect(screen.getByRole('button', { name: 'Back' })).toBeVisible();
```

Comment:
Just curious, why `toBeVisible` over `toBeInTheDocument` for elements that we always show?